Revisions

huddlej opened the pull request:
Commits address revisions requested by reviewers.

huddlej committed:
Change references of "genome" to "gene sequence"

Corrects a misnomer in the text caused by our informal use of the word
"genome".

joverlee521 committed:
Update Fig2 to include diversity panel

Includes the new screenshot that has the diversity panel and updates
the Figure 2 caption.

joverlee521 committed:
Update Results 3.1 with diversity panel interaction

joverlee521 committed:
Results 3.3: remove "entropy"

Replace "entropy" with "diversity" to avoid confusion and remove intro
to diversity panel since it has been introduced in Results 3.1.

jameshadfield committed:
[typesetting] change url to href

Allows us to drop the protocol from rendered links in the text

jameshadfield committed:
Add URL to data viz in Implementation section

Reviewer request

jameshadfield committed:
Clarify relationship between nextstrain and nextflu

As requested by review

huddlej committed:
Add more details to methods

Adds specific tools used to infer and annotate the time tree and
includes details about how we defined clades.

huddlej committed:
Clarify definition of "clade"

huddlej committed:
Clarify that Auspice is the visualization platform

huddlej committed:
Note shared first authorshop in header

huddlej committed:
Simplify title and clarify running title

Changes the title to replace the jargon phrase "vaccine strain selection"
with "vaccine composition" and expands the running title slightly to be
more descriptive.

huddlej committed:
Replace "strain" with "virus"

"Strain" is yet another jargon term that we do not really need in this
context and do not already define in the text. Use "virus" throughout
instead to simplify the text.

jameshadfield committed:
add background visualisation references

huddlej committed:
Rewrite the introduction

Clarifies the process of vaccine selection and the terminology used to
describe it.

jameshadfield committed:
comment on data sharing

jameshadfield commented on manuscript/measurements-panel.tex:
```diff
@@ -110,11 +116,13 @@ \section{Introduction}
 Additionally, this view shows a summary statistic instead of the underlying distributions of the data, concealing the number and variance of measurements for each reference virus.
 
+To overcome the limitations of existing antigenic visualizations, we applied user-driven design based on the goals of decision makers described above and used standard visual design principles to produce a more expressive and effective visualization for influenza virologists.
+Below, we describe the resulting measurements panel, a new component of Nextstrain's interactive phylogenetic visualization platform, and provide two case studies that demonstrate the practical value of the measurements panel for vaccine composition decisions.
```
:+1: this is a good framing sentance

huddlej committed:
Explicitly name "the measurements panel"

Refines the last paragraph of the introduction to state the name of the
tool we present here and avoid confusion with Nextstrain or nextflu.

huddlej committed:
Simplify introduction a bit more

Drops the paragraph on long-term antigenic evolution (which is outside
the scope of this paper) in favor of a couple sentences leading into our
approaches to phylogenetic visualizations of antigenic evolution. Uses
the references from the original paragraph (Felsenstein, Lemey), adds a
reference to Smith et al. 2004, and adds a reference to Steinbruck et
al. as part of recent work in this area.

huddlej committed:
Clarify the data sources for visualizations

Conceptually links phylogenetic and antigenic visualizations with
earlier paragraphs through "HA gene sequences" and "HI assays".

joverlee521 committed:
Move demo URL to introduction per reviewer request

jameshadfield committed:
fix formatting of long url

huddlej committed:
Move public data statement to data availability
